packages/blocks/src/_common/components/rich-text/virgo/nodes/link-node/link-popup/link-popup.ts

```diff
@@ -81,6 +81,20 @@ export class LinkPopup extends WithDisposable(LitElement) {
     );
   }
 
+  protected override firstUpdated() {
+    if (!this.linkInput) return;
+
+    this._disposables.addFromEvent(this.linkInput, 'copy', e => {
```

Comment:
@Saul-Mirone do we have some framework encapsulation for this kind of logic?

Reply:
To resolve this issue, migrating the link popup to a portal-style format can be a good solution.

Reply:
We don't have it for now. Maybe we can have something like `prevent events` config in the future. But it will be a low priority task to me because this pattern is rare for now. We can give it a consider if logic like this grows rapidly.